src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -853,27 +849,20 @@ private Compilation CreateEmptyCompilation()
                 return null;
             }
 
-            public Task<bool> HasSuccessfullyLoadedAsync(
+            public async ValueTask<bool> HasSuccessfullyLoadedAsync(
```

Comment:
This should probably remain `Task<bool>`, as the performance will be slightly better than `ValueTask<bool>` in all cases except where you are using `IValueTaskSource` (not applicable here).